HW_11_return_number.py

```diff
@@ -0,0 +1,23 @@
+import os
+os.system('cls')
+
+
+def is_function_result_num(func):
+    def wrapper(*args):
+        result = func(*args)
+        if not isinstance(result, (float, int)):
+            print(f'{result} -- Результат функции НЕ ЯВЛЯЕТСЯ числом!')
+        else:
+            print(f'{result} -- Результат функции является числом.')
```

Comment:
`is_function_result_num` decorator name, so I would expect to return True or False

```suggestion
        if not isinstance(result, (float, int)):
            HERE
        else:
            AND HERE
```

Reply:
Done